src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -343,7 +311,29 @@ def add_to_nwbfile(
                 nwbfile.add_device(device)
             image_series_kwargs["device"] = device
 
-        timing_type = self.get_timing_type()
+        if always_write_timestamps:
+            timestamps = self._timestamps if self._timestamps is not None else self.get_timestamps()
```

Comment:
```suggestion
            timestamps = self.get_timestamps()
```

self.get_timestamps already checks if self._timestamps is None, so this should be the same but simpler.

Reply:
Yes, thanks.